packages/playground-examples/copy/ja/TypeScript/Primitives/Unknown and Never.ts

```diff
@@ -1,14 +1,14 @@
-// Unknown
+// unknown型
 
-// Unknown is one of those types that once it clicks, you
-// can find quite a lot of uses for it. It acts like a sibling
-// to the any type. Where any allows for ambiguity - unknown
-// requires specifics.
+// unknown型は、非常に多くの箇所で使用されています。
+// unknown型は、any型とよく似た振る舞いをします。
+// 違いは、anyは曖昧なままでも使用できますが、 unknownは
+// 詳細を必要とする点です。
 
-// A good example would be in wrapping a JSON parser. JSON
-// data can come in many different forms and the creator
-// of the json parsing function won't know the shape of the
-// data - the person calling that function should.
+// JSONパーサーのWrapperは良い例です。
+// JSONのデータは、多種多様であるため、JSONパーサーの作者は、
+// データがどのような形になっているのかを知ることはできません。
+// データの形は、パーサーを呼び出す利用者が知っているべきなのです。
```

Comment:
このshouldは「べき」ではなく、json parserを使う人間は「当然」知っているdataの形をjson parserの提供者は知ることができないという対比を強調する意味だと思います。

```suggestion
// JSONのデータは多種多様であるため、JSONパーサーの利用者が
// データの形を知っているにもかかわらず、JSONパーサーの作者は、
// データがどのような形になっているのかを知ることはできません。
```